molgenis-data-i18n/src/main/java/org/molgenis/data/i18n/LanguageServiceImpl.java

```diff
@@ -18,40 +17,27 @@
 import static org.molgenis.security.core.runas.RunAsSystemAspect.runAsSystem;
 
 @Service
-public class LanguageService
+public class LanguageServiceImpl implements LanguageService
```

Comment:
Use interface where possible instead of the Impl

Reply:
Done